OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Hi Avi,

As suspected, docker stop isn't converted to a forceShutdown request for WLS server. Here is what I tried:
docker pull container-registry.oracle.com/middleware/weblogic:12.2.1.1
docker run container-registry.oracle.com/middleware/weblogic:12.2.1.1

And then on the side I issued a docker stop request using a generated container name

docker stop epic_meitner

I can see that shutdown hook is not invoked, and the container stopped after 10 seconds (the default difference between the time SIGTERM and SIGKILL are sent to the container process)

Had it worked, I would have expected to see something like the following on stdout of the container:

<Jun 5, 2017 3:18:30,533 PM PDT> <Notice> <WebLogicServer> <BEA-000388> <JVM called the WebLogic Server shutdown hook. The server will force shutdown now.> 
<Jun 5, 2017 3:18:30,534 PM PDT> <Notice> <WebLogicServer> <BEA-000396> <Server shutdown has been requested by <WLS Kernel>.> 
<Jun 5, 2017 3:18:30,538 PM PDT> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FORCE_SUSPENDING.> 
<Jun 5, 2017 3:18:30,549 PM PDT> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to ADMIN.> 
<Jun 5, 2017 3:18:30,549 PM PDT> <Notice> <WebLogicServer> <BEA-000365> <Server state changed to FORCE_SHUTTING_DOWN.> 
<Jun 5, 2017 3:18:30,561 PM PDT> <Notice> <Log Management> <BEA-170037> <The log monitoring service timer has been stopped.> 
<Jun 5, 2017 3:18:30,601 PM PDT> <Warning> <JMX> <BEA-149513> <JMX Connector Server stopped at service:jmx:iiop://10.147.117.136:7001/jndi/weblogic.management.mbeanservers.edit.> 
<Jun 5, 2017 3:18:30,602 PM PDT> <Warning> <JMX> <BEA-149513> <JMX Connector Server stopped at service:jmx:iiop://10.147.117.136:7001/jndi/weblogic.management.mbeanservers.domainruntime.> 
<Jun 5, 2017 3:18:30,666 PM PDT> <Warning> <JMX> <BEA-149513> <JMX Connector Server stopped at service:jmx:iiop://10.147.117.136:7001/jndi/weblogic.management.mbeanservers.runtime.> 
Stopping Derby server...
Derby server stopped.